config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BuilderUtils.java

```diff
@@ -9,13 +9,15 @@
 import com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder.EntityBuilderException;
 import com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle.loader.DependencyBundleLoadException;
 import org.apache.commons.collections4.MapUtils;
+import org.jetbrains.annotations.NotNull;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Map.Entry'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=76&issues=AWZZr2vN9tx2us2yAaRl&open=AWZZr2vN9tx2us2yAaRl)

[ISSUE_KEY]: AWZZr2vN9tx2us2yAaRl